.evergreen/config/test-variants.yml

```diff
@@ -19,7 +19,7 @@ buildvariants:
       - ".sharded .local !.3.6 !.4.0 !.4.2 !.4.4 !.5.0"
       - ".loadbalanced .local !.3.6 !.4.0 !.4.2 !.4.4 !.5.0"
       - "test-atlas-connectivity"
-      - ".ocsp !.4.4"
+      - ".ocsp !.4.4 !.5.0"
```

Comment:
IIUC, this drops OCSP testing on 5.0. What the impetus for this change? The [OCSP spec test README](https://github.com/mongodb/specifications/blob/master/source/ocsp-support/tests/README.rst) doesn't talk about server versions.